src/layout/TheSidebar.vue

```diff
@@ -34,7 +34,7 @@
             >
             <textarea
               aria-labelledby="notitieblok"
-              v-focus
+              id="cm-notitieblok"
```

Comment:
There is, but I would have to either pass it as a prop to whatever component wants it to focus, or inject it, which feels like it would be more work than the vanilla way.